documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -21,6 +21,27 @@ for a complete description of all its capabilities and configuration options,
 refer to the Infinispan project documentation at
 http://infinispan.org/documentation/[infinispan.org].
 
+=== Why use Hibernate OGM with Infinispan?
+
+Infinispan offers a lot in term of scalability and elasticity but
+it can have a steep learning curve.
+
+If you ar already familiar with the JPA API you will
+be able to store your data in Infinispan quickly and you will also benefit
+out-of-the-box from the optimizations that a framework like Hibernate OGM can
+apply under the hood.
+
+In particular:
+
+- no need to learn about Protobuf and Externalizer encoding format
```

Comment:
I'm afraid this isn't true. The user will eventually have to face learning about `Protobuf` formats at least.

I would frame it as "you can get started without having to learn Protobuf first; it will still be useful to learn Protobuf to maintain your applications".